servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/NamespaceInfoSerializer.java

```diff
@@ -96,10 +93,14 @@ private static NamespaceInfo parseFromEntry(Node xEntry) {
                                             namespaceInfo.setNamespaceSku(NamespaceSku.Unknown);
                                         }
                                         break;
+                                    default:
```

Comment:
[SpotBugs-P2]:
This method contains a switch statement where default case is missing. Usually you need to provide a default case.